pkg/cmd/server/origin/master.go

```diff
@@ -152,7 +152,7 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 		extra = append(extra, i.InstallAPI(open)...)
 	}
 
-	handler = indexAPIPaths(handler)
+	handler = indexAPIPaths(c.Options.APILevels, configapi.GetEnabledAPIVersionsForGroup(*c.Options.KubernetesMasterConfig, configapi.APIGroupKube), handler)
```

Comment:
> Why does that method need an instance of the master config, instead of a pointer to it? Seems like a huge object to be copying for this use.

It's a pretty tiny object, already available locally, that is copied once per startup.  The copy is fine.  And yeah, that's a pointer because its allowed to be nil.